styles/popUp.css

```diff
@@ -0,0 +1,163 @@
+.popup {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 90%;
+  background-color: #fff;
+  padding: 1.6rem;
+  border-radius: 15px;
+  box-shadow: 0 3rem 5rem rgba(0, 0, 0, 0.3);
+  z-index: 10;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 2rem;
+  overflow: hidden;
+}
+
+.popup__img {
+  width: 100%;
+}
+
+.popup__left {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+}
+
+.popup__exit {
+  position: absolute;
+  right: 3rem;
+  top: 2.5rem;
+  cursor: pointer;
+  border: none;
+  outline: none;
+}
+
+.popup__name {
+  font-size: 3.2rem;
+  font-weight: 600;
+  color: #172b4d;
+}
+
+.popup__canopy > * {
+  margin-top: 0;
+}
+
+.popup__canopy-list {
+  margin: 0;
+}
+
+.popup__tech {
+  justify-content: flex-start;
+  gap: 2rem;
+  margin-left: 3px;
+  margin-top: 1.2rem;
+}
+
+.popup__description {
+  font-size: 1.6rem;
+  color: #344563;
+  line-height: 2.4rem;
+  margin-left: 0.4rem;
+  width: 100%;
+}
+
+.popup__buttons {
+  width: 100%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-top: 4.9rem;
+}
+
+.popup__buttons > a {
+  display: flex;
+  width: 16rem;
+  height: 5rem;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.popup__buttons img {
+  position: relative;
+  padding-inline: 0.5rem;
+}
+
+.overlay {
+  position: fixed;
+  inset: 0;
+  background-color: #ebecf0;
+  z-index: 5;
+  opacity: 0.8;
+  filter: blur(10px);
+}
+
+.hidden {
+  display: none;
+}
+
+.popup-open {
+  overflow: hidden;
+}
+
+@media screen and (min-width: 768px) {
+  .popup {
+    width: 50%;
+    height: 95%;
+    gap: 0;
+    padding: 2.4rem;
+    overflow: visible;
+  }
```

Comment:
- [ ] Please try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=34-1143&t=wS8Tv8oWzsuxkB9V-0). The popup covers at least 80% of the screen size. Kindly implement this, in order to have a professional and user-friendly portfolio ☺️.

Your design 😎 
![image](https://github.com/ahmedeid6842/Portfolio/assets/99394374/5215844d-d167-4ab9-8ef1-ab800883491c)

Figma design 😎 
![image](https://github.com/ahmedeid6842/Portfolio/assets/99394374/4fdf2e02-ac82-4cc3-986d-5153d72d0088)